spec/ginkgo_spec.vim

```diff
@@ -0,0 +1,73 @@
+source spec/support/helpers.vim
+
+describe "Ginkgo"
+
+  before
+    let g:test#go#runner = 'ginkgo'
```

Comment:
Could you  please add a test where is not set to test the search based runner detect